extensions/GitExtension/FileExplorerGitIntegration.UnitTest/WslIntegratorUnitTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using FileExplorerGitIntegration.Models;
+
+namespace FileExplorerGitIntegration.UnitTest;
+
+[TestClass]
+public class WslIntegratorUnitTests
+{
+    [TestMethod]
+    [DataRow("\\wsl$\\Ubuntu-20.04\\home\\user\\repo")]
```

Comment:
Validate that combinations of upper and lower case of `\\wsl$` and `\\wsl.localhost` prefixes also parse correctly. We should be case-insensitive on the part of that path that is non-Linux and doesn't refer to a distro name.

Reply:
Made test modifications in latest changes. Thanks!